assets/queries/k8s/limit_capabilities_from_the_container/test/positive_expected_result.json

```diff
@@ -1,6 +1,6 @@
 [
 	{
-		"queryName": "Limit Capabilities Of The Container",
+		"queryName": "Not Limited Capabilities For Container",
```

Comment:
Unlimited?

Reply:
Prefer the use of Not Limited in this case